cpp/util/etcd.cc

```diff
@@ -474,6 +476,16 @@ void EtcdClient::WatchInitialGetDone(WatchState* state, GetResponse* resp,
   // TODO(pphaneuf): Need better error handling here. Have to review
   // what the possible errors are, for now we'll just retry after a delay.
   if (!task->status().ok()) {
+    if (task->status().error_code() == util::error::UNAVAILABLE) {
```

Comment:
I think this should never happen, since this is called when the `Get` is done (which goes through `GenericRequest`, and thus, `FetchDone`).

I'm also seeing the retry just below what you've added, and considering that it's also duplicate, seems sketchy!